schema.sql

```diff
@@ -1,9 +1,10 @@
 
 CREATE TABLE animals (
   id INT NOT NULL PRIMARY KEY,
-  name VARCHAR(100) NOT NULL,
+  name VARCHAR(50) NOT NULL,
   date_of_birth DATE NOT NULL,
   escape_attempts INT NOT NULL,
   neutered BOOLEAN NOT NULL,
-  weight_kg NUMERIC(4, 2) NOT NULL
+  weight_kg NUMERIC(5, 2) NOT NULL,
+  species VARCHAR(50)
```

Comment:
- [ ] you included the species column in the original `CREATE TABLE` however you are expected to update the existing table by doing `ALTER TABLE animals ADD species varchar(255);`

Reply:
Hello Mike, I have added the species column to the animals table.